src/vs/workbench/contrib/terminalContrib/accessibility/browser/terminalAccessibleBuffer.ts

```diff
@@ -109,9 +115,15 @@ export class AccessibleBufferWidget extends DisposableStore {
 				this._editorWidget.updateOptions({ fontFamily: font.fontFamily, fontSize: font.fontSize, lineHeight: font.charHeight ? font.charHeight * font.lineHeight : 1, letterSpacing: font.letterSpacing });
 			}
 		}));
-		this.add(this._xterm.raw.onWriteParsed(async () => {
+		this.add(this._xterm.raw.onScroll(async () => {
 			if (this._isActive()) {
-				await this._updateEditor(true);
+				await this._updateEditor();
```

Comment:
Even better than checking if it's active would be to only register the listeners when it's active and then dispose then when it becomes inactive. That means less no-op event listeners firing and we can avoid this _isActive check and use runAndSubscribe (`Event.any(this._xterm.raw.onScroll, this._xterm.raw.onWriteParsed)` will probably work for xterm.js events)

Reply:
the Event.any scares me because they'll probably happen simutaneously.